docs/configs/html/conf.py

```diff
@@ -175,7 +175,7 @@
 .. _installer: https://github.com/digital-asset/daml/releases/download/v{release}/daml-sdk-{release}-windows.exe
 .. _protobufs: https://github.com/digital-asset/daml/releases/download/v{release}/protobufs-{release}.zip
 .. _api-test-tool: https://repo1.maven.org/maven2/com/daml/ledger-api-test-tool/{release}/ledger-api-test-tool-{release}.jar
-.. _http-json: https://github.com/digital-asset/daml/releases/download/v{release}/http-json-{release}.jar
```

Comment:
`http-json` doesn't seem to be used anywhere. That's probably why it wasn't an issue that it was missing on the PDF config.